src/webgpu/api/validation/render_pass/attachment_compatibility.spec.ts

```diff
@@ -206,13 +206,13 @@ g.test('render_pass_and_bundle,color_count')
   .fn(t => {
     const { passCount, bundleCount } = t.params;
     const bundleEncoder = t.device.createRenderBundleEncoder({
-      colorFormats: range(bundleCount, () => 'rgba8unorm'),
+      colorFormats: range(bundleCount, () => 'rgba8uint'),
```

Comment:
Question: is the reason behind `rgba8unorm` -> `rgbau8uint` being `rgba8unorm` has a render target pixel byte cost of 8 ( > 4) that could possibly go beyond the limit in some params setting?

Reply:
Yup! Basically this test is testing formats matching, so I chose to use a format that even with the max number of attachments (8), we won't exceed the max sample bytes. (Max sample bytes is 32, so 4*8 <= 32, while 8*8 > 32).